docsource/modules150-160.rst

```diff
@@ -156,7 +156,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | google_calendar                                 |Done                  |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| google_drive                              |                      |                                                 |
+| |del| google_drive                              |Deprecated module     |Obsolete module, features completely removed     |
```

Comment:
```suggestion
| |del| google_drive                              | Deprecated module    | Obsolete module                                  |
```
I prefer it shorter.

Reply:
Same like https://github.com/OCA/OpenUpgrade/pull/4002